IPython/core/display.py

```diff
@@ -27,7 +27,8 @@
 'display_javascript', 'display_pdf', 'DisplayObject', 'TextDisplayObject',
 'Pretty', 'HTML', 'Markdown', 'Math', 'Latex', 'SVG', 'JSON', 'GeoJSON', 'Javascript',
 'Image', 'clear_output', 'set_matplotlib_formats', 'set_matplotlib_close',
-'publish_display_data', 'update_display', 'DisplayHandle', 'Video']
+'publish_display_data', 'update_display', 'DisplayHandle', 'Video', 'get_repr_mimebundle',
+'RecursiveObject', 'ReprGetter']
```

Comment:
Fair enough for it to be clean, but I still expect people to need to register formatter for `RecursiveObject` if they want nice representation for format it does not support currently, so you need a handle on it.  ReprGetter also need to be public if you are computing recursive Repr on multiple task/threads